lib/still/preprocessor.ex

```diff
@@ -148,13 +155,18 @@ defmodule Still.Preprocessor do
   end
 
   defp preprocessors do
-    Map.merge(@default_preprocessors, user_defined_preprocessors())
+    preprocessors_to_list(user_defined_preprocessors()) ++
+      preprocessors_to_list(@default_preprocessors)
```

Comment:
I think you can just to `Enum.concat(user_defined_preprocessors(), @default_preprocessors)`

Reply:
👍